Lib/test/test_platform.py

```diff
@@ -762,13 +762,14 @@ def invoke_platform(self, *flags):
             platform._main(args=flags)
         return output.getvalue()
 
+    @support.force_not_colorized
     def test_unknown_flag(self):
+        output = io.StringIO()
         with self.assertRaises(SystemExit):
-            output = io.StringIO()
             # suppress argparse error message
             with contextlib.redirect_stderr(output):
                 _ = self.invoke_platform('--unknown')
-            self.assertStartsWith(output, "usage: ")
```

Comment:
The relevant issue is that `self.assertStartsWith` will ordinarily never be called due to the `SystemExit`. 

@donBarbos in the future, please explain and provide context for your PRs, rather than just saying 'fix' -- assume reviewers are looking at your proposed changes with no prior context.

A